---
layout: page
title: Projects
permalink: /projects/
---

**Project 1: Sibling GWAS**

There is increasing evidence that genetic effects on social phenotypes are substantially attenuated after controlling for family fixed effects in samples of siblings. For example, [Lee et al., 2018](https://www.nature.com/articles/s41588-018-0147-3) found that the effects of the genome-wide significant SNPs for years of education attenuated by around one half. Consistent with this, [Kong et al (2018)](https://science.sciencemag.org/content/359/6374/424) found substantial evidence that non-inherited genetic variants were associated with outcomes in offspring. These findings suggest that genetic effects estimated in GWAS for some (particularly social) phenotypes may be biased. In this project, we will conduct GWA on samples of siblings using a within family model (i.e. including a family fixed effect). This analysis will be conducted across a broad range of available phenotypes (e.g. education, cognitive performance, BMI, height, CRP, etc) to investigate the prevalence of “within family” bias in established GWAS of unrelated individuals. This project will focus on a sibling design, but there are plans to expand future projects to include mother-father-offspring trio designs.

Research question: Do total, between and within family SNP-phenotype associations differ?

We are conducting GWAS of 29 phenotypes using data from studies with siblings and relevant phenotypes. A full description of the analysis can be found in the [statistical analysis plan](https://docs.google.com/document/d/1A9HbwdVEnSul7eSeTdYfLtH1rkjTv5VMYVBs0wfZnxQ/edit?usp=sharing). The end of the SAP lists the studies that are contributing to the analysis, study eligibility requirements and the phenotypes that are currently included in the study.

**Code**

The analysis is being run independently by analysts for each study. Laurence Howe has written an automated pipeline that is available on his [GitHub page](https://github.com/LaurenceHowe/SiblingGWAS). This automates most of the process of running the analysis, creating the study summary data and packages summary associations (SNP-phenotype associations) up for upload to University of Bristol servers. We will meta-analyze the summary data across the studies.

If you are interested in being involved in this project please contact [Neil Davies](mailto:neil.davies@bristol.ac.uk) and [Laurence Howe](mailto:laurence.howe@bristol.ac.uk).

